plugins/de.cau.cs.kieler.klighd.ui/src/de/cau/cs/kieler/klighd/ui/printing/dialog/PrinterBlock.java

```diff
@@ -95,21 +95,29 @@ public static Group createContents(final Composite parent, final DataBindingCont
         final Realm realm = bindings.getValidationRealm();
 
         // Observable to display the printer resp. file name, if print to file is set.
-        final ComputedValue computedValue = new ComputedValue(realm) {
+        final ComputedValue<?> computedValue = new ComputedValue<>(realm) {
```

Comment:
For me this replaces the warning with an error: "'<>' cannot be used with anonymous classes"

Reply:
fixed in 145ba1a33fc32914952fc95d94f95a533f2871f5